drivers/npmx/npmx_shell.c

```diff
@@ -448,10 +462,18 @@ static int cmd_charger_trickle_get(const struct shell *shell, size_t argc, char
 	npmx_error_t err_code = npmx_charger_trickle_voltage_get(charger_instance, &voltage);
 
 	if (check_error_code(shell, err_code)) {
-		shell_print(shell, "Value: %d mV.", npmx_charger_trickle_convert_to_mv(voltage));
+		uint32_t voltage_mv;
+
+		if (npmx_charger_trickle_convert_to_mv(voltage, &voltage_mv)) {
+			shell_print(shell, "Value: %d mV.", voltage_mv);
+		} else {
+			shell_error(
+				shell,
+				"Error: unable to convert trickle voltage value to mv.");
+		}
 	} else {
 		shell_error(shell,
-			    "Error: unable to read battery normal termination voltage value.");
+			    "Error: unable to read trickle voltage value.");
```

Comment:
@nordic-piks @nordic-segl Does this change affect CI?

Reply:
Seems not, as we do not have test case.
What are the conditions when read can fail?
What are the condition when conversion of value get from pmic can fail?

Reply:
When it comes to SET commands:
- correct value - we search for "Success: <value>" and we confirm that <value> is equal to what we wanted to set;
- invalid value - here we check error messages.

In all GET commands we search for "Success: <value>" and we confirm that <value> is equal to what we expect.
There is no negative testing for GET because we are unable to "inject" invalid value.